vsc_pdf_template_transformer/lib/utils/evaluator.dart

```diff
@@ -88,3 +89,84 @@ List<T>? evaluateList<T>(dynamic expression, Map<String, dynamic> data) {
 
   return List<T>.from(result, growable: false);
 }
+
+T? evaluateEnum<T extends Enum>(
+    List<T> values, dynamic expression, Map<String, dynamic> data) {
+  final result = _evaluateDynamic(expression, data);
+  if (result == null) {
+    return null;
+  }
+
+  return values.byName(result.toString());
+}
+
+Font? evaluateFont(dynamic expression, Map<String, dynamic> data) {
+  final fontEnum = evaluateEnum(Type1Fonts.values, expression, data);
+  if (fontEnum == null) {
+    return null;
+  }
+
+  return Font.type1(fontEnum);
+}
+
+List<Font> evaluateFontList(
+    List<dynamic>? expressions, Map<String, dynamic> data) {
+  final List<Font> list = [];
+  if (expressions != null && expressions.isNotEmpty) {
+    for (final e in expressions) {
+      final result = evaluateFont(e, data);
+      if (result != null) {
+        list.add(result);
+      }
+    }
+  }
+  return list;
+}
+
+TextDecoration? evaluateTextDecoration(
+    dynamic expression, Map<String, dynamic> data) {
+  final result = _evaluateDynamic(expression, data);
+  if (result == null) {
+    return null;
+  }
+  // TODO Should accept a list or string. If list, use TextDecoration.combine()
```

Comment:
We should support a List or String for `result`. If it is a list, each element is a string and should be combined using `TextDecoration.combine()`.